zwave_js_server/util/lock.py

```diff
@@ -143,12 +143,7 @@ async def set_usercode(
 ) -> SetValueResult | None:
     """Set the usercode to index X on the lock."""
     value = get_code_slot_value(node, code_slot, LOCK_USERCODE_PROPERTY)
-    usercode = str(usercode)
-
-    if len(usercode) < 4:
-        raise ValueError("User code must be at least 4 digits")
-
-    return await node.async_set_value(value, usercode)
+    return await node.async_set_value(value, str(usercode))
```

Comment:
Do we need to copy the usercode to string? The parameter is already typed as a string.

Reply:
this may have been a workaround to address upstream issues. Typing is right, and if there are issues they should probably be fixed upstream